src/main/java/rolling/RollOutput.java

```diff
@@ -0,0 +1,40 @@
+package rolling;
+
+import org.javacord.api.entity.message.embed.EmbedBuilder;
+
+import java.util.List;
+
+public class RollOutput {
+    private final List<EmbedBuilder> embeds;
+    private final boolean plotDiceUsed;
+    private final int rollTotal;
+    private final boolean opportunity;
+
+    public RollOutput(List<EmbedBuilder> embeds, boolean plotDiceUsed, int rollTotal, boolean opportunity) {
```

Comment:
Really should be avoiding mixing more discord code into the rolling classes.

Reply:
I should, but not sure if it's necessary for a class that's just supposed to old the output of a function instead of using tuples